app/views/events/index.html.erb

```diff
@@ -3,11 +3,7 @@
 </style>
 <p id="notice"><%= notice %></p>
 
-<% if @admin %>
```

Comment:
Can we keep if someone is logged in as an admin they can still see who is logged in?